docs/content/latest/admin/yb-ctl.md

```diff
@@ -16,78 +16,197 @@ showAsideToc: true
-  --verbose             If specified, will log internal debug messages to
-                        stderr.
-  --install-if-needed   With this option, if YugabyteDB is not yet installed
-                        on the system, the latest version will be downloaded
-                        and installed automatically.
-```
-
-Here are the default values for all the optional arguments.
-
-| Optional argument                | Default                               | Description                                                  |
-| -------------------------------- | ------------------------------------- | ------------------------------------------------------------ |
-| `--binary_dir`                   | Same directory as the `yb-ctl` binary | Location of the `yb-master` and the `yb-tserver` binaries    |
-| `--data_dir`                     | `/yugabyte-data`         | Location of the data directory for YugabyteDB           |
-| `--replication_factor` or `--rf` | `1`                                   | Number of replicas for each tablet, should be an odd number (e.g. `1`,`3`,`5`) so that majority consensus can be established |
-| `--require_clock_sync`           | `false`                               | Tells YugabyteDB whether to depend on clock synchronization between the nodes in the cluster |
-| `--num_shards_per_tserver`       | `2`                                   | Number of shards (tablets) per tablet server for each table  |
-
-## Create a cluster
+yb-ctl [ command ] [ argument, argument2, ... ]
+```
+
+## Commands
+
+### create
+
+Creates a local cluster.
+
+For more details and examples, see:
+
+- [Create a local cluster](#create-a-local-cluster)
+- [Create a cluster across multiple zones, regions, and clouds](#Create-a-cluster-across-multiple-zones-regions-and-clouds)
+- [Create a cluster with custom flags](#Create-a-cluster-with-custom-flags).
+
+### start
+
+Starts the existing cluster or, if not existing, creates and starts the cluster.
+
+### stop
+
+Stops the cluster, if running.
+
+### destroy
+
+Destroys the current cluster.
+
+For details and examples, see [Destroy a local cluster](#destroy-a-local-cluster).
+
+### status
+
+Display the current status of the cluster.
+
+For details and examples, see [Check cluster status](#check-cluster-status).
+
+### restart
+
+Restarts the current cluster all at once.
+
+For details and examples, see:
+
+- [Restart a cluster](#restart-a-cluster)
+- [Restart with custom tags](#restart-with-custom-tags)
+
+### wipe_restart
+
+Stops the current cluster, wipes all data files and starts the cluster as before (losing all flags).
+
+For details and examples, see [Wipe and restart with placement info flags](#wipe-and-restart-with-placement-info-flags).
+
+### add_node
+
+Adds a new node to the current cluster.
+
+For details and examples, see:
+
+- [Add nodes](#add-nodes)
+- [Create a cluster across multiple zones, regions, and clouds](#create-a-cluster-across-multiple-zones-regions-and-clouds)
```

Comment:
While not obvious from the title, the `add_node` command is used a couple of times in that example and may be useful. My goal was to link to relevant examples. This is planned to be an iterative process of incremental improvements. After reviewing the examples, it looks like many of them might be better to be included in the command section, rather than in separate example sections — some are rather short. In the end, that particular example probably belongs somewhere else, in the development or deployment sections.